src/functionalities.js

```diff
@@ -0,0 +1,120 @@
+import updateStatus from './status.js';
+
+export default class Tasks {
+  constructor() {
+    this.tasksList = [];
+  }
```

Comment:
The constructor here is totally unused Code. You Aren't Gonna Need It **(YAGNI)**. this will ensure you also keep it simple and stupid **(KISS)**

Reply:
The constructor function here is necessary when there isn't a tasks  object in the `localStorage`. Otherwise, when adding the first task, it won't be added, and will get an error instead.